### DISCUSSION TITLE
How to disabled examine umbraco version 8.5.4

### DISCUSSION THREAD

**Discussion by mitraganesh** [umbraco/Umbraco-CMS#12737] [Features and ideas]
I want to disable examine indexing due lock issue on Azure and never used it in front end as well as back end. Is it possible?
How to do this?

I search over forum but not found any suitable solution. Also asked  in forum but no body help.

**Comment by TwoMoreThings** [comment]
I think you can only do this on the front end servers in a load balanced environment. See this link for details.
https://shazwazza.com/post/can-i-disable-examine-indexes-on-umbraco-front-end-servers/

You can get it to work in a load balanced environment without locking issues. We have one back end editing server and 4 delivery instances in Azure. It was mostly following the docs. We are, however, using 8.16 and I think there were some bugs that required at least 8.13, but I can't be 100% certain, but you should definitely upgrade.

HTH, tom
